lib/ui/index.js

```diff
@@ -79,6 +79,10 @@ UI.prototype.writeLine = function(data, writeLevel) {
   if (this.writeLevelVisible(writeLevel)) {
     this.write(data + EOL);
   }
+
+  if (/ERROR/i.match(writeLevel)) {
```

Comment:
I think this section needs to be tweaked a bit.  Won't this write the output to both `this.outputStream` (from the line directly above) and to `this.errorStream`?